src/tabnote.ts

```diff
@@ -130,7 +130,7 @@ export class TabNote extends StemmableNote {
   }
 
   protected ghost: boolean;
-  protected glyphPropsArr: GlyphProps[] = [];
+  protected fretEl: Element[] = [];
```

Comment:
I think this should either be `fret` or `fretElement`. I don't think the name `fretEl` is consistent with the other times you've included an Element object as an instance var.

Reply:
done